script/hassfest/mypy_config.py

```diff
@@ -36,8 +36,9 @@
     "plugins": "pydantic.mypy",
     "show_error_codes": "true",
     "follow_imports": "normal",
-    "enable_incomplete_feature": ",".join(  # noqa: FLY002
+    "enable_incomplete_feature": ", ".join(  # noqa: FLY002
         [
+            "InlineTypedDict",
```

Comment:
I don't think we show enable this, mostly because of the release notes @ pyright:

![CleanShot 2024-07-07 at 14 55 49@2x](https://github.com/home-assistant/core/assets/195327/b4dffe61-3391-4b78-b45f-364a729edbbc)